api/v1alpha1/openapi.yaml

```diff
@@ -3031,6 +3031,63 @@ components:
               description: 'The directory in which the executable will be run from if it is left empty it will run from the users home directory.'
           required:
             - run
+    DeviceUpdatePolicySpec:
+      type: object
+      properties:
+        downloadSchedule:
+          $ref: '#/components/schemas/DownloadPolicy'
+        updateSchedule:
+          $ref: '#/components/schemas/UpdatePolicy' 
+    DownloadPolicy:
+      allOf:
+        - $ref: '#/components/schemas/UpdateSchedule'
+      description: "A policy defining the download schedule."
+    UpdatePolicy:
+      allOf:
+        - $ref: '#/components/schemas/UpdateSchedule'
+      description: "A policy defining the update schedule."
+    UpdateSchedule:
+      type: object
+      properties:
+        timeZone:
+          type: string
+          description: >
+            Time zone identifiers follow the IANA format AREA/LOCATION, where AREA
+            represents a continent or ocean, and LOCATION specifies a particular
+            site within that area.  e.g., America/New_York, Europe/Paris.
+            Only unambiguous 3-character time zones are supported ("GMT", "UTC").
+          default: "Local"
```

Comment:
For reference, k8s does not consider "Local"[1] valid.

[1] https://github.com/kubernetes/kubernetes/blob/776fb24e066d54cf84a52366aeea121fa5748e27/pkg/apis/batch/validation/validation.go#L834

Reply:
done